### DISCUSSION TITLE
dual extruder

### DISCUSSION THREAD

**Discussion by Podster2020** [supermerill/SuperSlicer#3838] [General]
i get an error when trying to print with dual extruders, changed the names of tools to T0 and T1 but klipper gives an error of extruder not configured, is T0 and T1 the wrong names as had same error when left blank

**Comment by salocain** [comment]
do you use klipper flavor ? 

if not, you need t0/t1/tx macro in you're klipper config

**Reply by Podster2020** [reply]
i do use klipper, 

should i be setting it up with some other way and not T0 T1?
